src/testdir/test_startup.vim

```diff
@@ -109,7 +109,9 @@ func Test_pack_in_rtp_when_plugins_run()
 endfunc
 
 func Test_help_arg()
-  CheckNotMSWindows
+  if !has('unix') && has('gui_running')
+    throw 'Skipped: does not work with gvim on MS-Windows'
+  endif
```

Comment:
Unrelated so you don't need to change.
If you have Windows, do you know why the following tests (or some of them?) hang on Windows: 
* Test_reverse
* Test_background_foreground
* Test_font
* Test_geometry
* Test_iconic

Those were disabled on Windows because CI was hanging but perhaps some of them can be enabled on Windows.

Reply:
I checked them.
Because the Windows version of gvim doesn't support the options such as `-background` and `-font`, an error message is shown. 

![image](https://user-images.githubusercontent.com/840186/111282408-503cd800-8681-11eb-9d6c-757e056d01ff.png)

None of the five tests works on Windows.

Reply:
should we add that as an comment to the tests, that the windows version does not support those arguments?

Reply:
Thanks @k-takata .  
From vim documentation `:help -font`, `:help -geometry`, `:help -background` (etc.) it was not indicated that these are not supported on Windows.

Reply:
These texts are written in gui_x11.txt, so they are for X11 only.